alpine-rails-buildkite/Dockerfile

```diff
@@ -0,0 +1,35 @@
+FROM dkubb/alpine-rails-nginx
+
+# Improve root state, intend to be pushed there
+WORKDIR /
+
+ENV RAILS_ENV=test                  \
+    PGBASE=/var/lib/postgresql      \
+    PGDATA=/var/lib/postgresql/data
+
+COPY etc /etc
+
+RUN rm -r /etc/sv/rails /etc/service/rails \
+ && /root/setup-directories.sh root r /etc/service/postgres /etc/service/buildkite \
+ && /root/setup-directories.sh postgres rw $PGBASE \
+ && ln -s -- /etc/sv/postgres /etc/service/postgres/run \
+ && ln -s -- /etc/sv/buildkite /etc/service/buildkite/run \
+ && echo '@testing-extra https://s3.amazonaws.com/alpine-packages/testing-extra' >> /etc/apk/repositories \
+ && apk add --update-cache imagemagick buildkite-agent@testing-extra phantomjs@testing-extra
+
+USER buildkite
+
+# Setup bundler for buildkite user
+RUN bundle config --global build.nokogiri '--use-system-libraries' \
```

Comment:
I don't think these are set globally per user. I think that they are set in the current directory you inhabit.

In the parent Dockerfile you'll notice I change the working directory `/opt/rails` and then run the command. It creates a `.bundle/config` file with the defaults.

My guess is since `/` is the current working directory it would be creating `/.bundle` with the config. I do not know if there is a way to set this in a truly global way.